docs/pages/installation.mdx

```diff
@@ -167,12 +154,14 @@ $ curl (=teleport.teleport_install_script_url=) | bash -s ${TELEPORT_VERSION?} $
 ### Package repositories
 
 The [one-line installation script](#one-line-installation-script) automatically
-detects a package manager on the local system and uses it to install Teleport
-from an upstream repository.
-
-If your system does not support the one-line installation script, read the
-instructions in this section for instructions on working with Teleport package
-repositories.
+installs Teleport from the official Teleport CDN.
+
+If your system does not support the one-line installation script or the
+`teleport-update` command, read the instructions in this section for
```

Comment:
A couple of paragraphs in the text mention the `teleport-update` command as an alternative to the one-line installation script. Should we have another subsection in the Linux section explaining `teleport-update`, e.g., after the "One-line installation script" subsection?